examples/index.html

```diff
@@ -26,28 +36,49 @@
         setRequestHeader(name, value) {
           // Do nothing.
         }
-        getResponseHeader(name, value) {
-          // Do nothing.
+        getResponseHeader(name) {
+          if (name === 'Content-Type' && focusedInput.classList.contains('json')) {
+            return 'application/json'
+          }
```

Comment:
As mentioned in https://github.com/github/auto-check-element/pull/18#pullrequestreview-266318385, https://github.com/github/auto-check-element/pull/18/files#diff-1fdf421c05c1140f6d71444ea2b27638R132 fails on `null.includes`.

```suggestion
          } else {
            return 'plain/text'
          }
```